src/modules/catalog.js

```diff
@@ -1584,6 +1584,109 @@ class Catalog {
     }
   }
 
+  /**
+   * Send patch delta catalog files to patch the current catalog
+   *
+   * @function patchCatalog
+   * @param {object} parameters - Additional parameters for catalog details
+   * @param {string} parameters.section - The section that you want to update
+   * @param {string} [parameters.notification_email] - An email address where you'd like to receive an email notifcation in case the task fails.
```

Comment:
Remove period at end of sentence as none of the other params use it

Reply:
Sorry copy-paste. Also removed from `updateCatalog`